Automattic/jetpack
# AI Assistant: Separate toolbar dropdown logic and UI

Part of #37015

## Proposed changes:

* Separates `AiAssistantToolbarDropdownContent` from its logic, allowing later reuse for the extension toolbar
* Renames files and folder so that `AiAssistantDropdown` isn't exported from `ai-assistant-controls`, avoiding confusion

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Use an extension, ask for a quick action like a translation, and use the "Ask AI Assistant" option
* Check that there is no regression
